pkg/collector/process_test.go

```diff
@@ -391,6 +395,10 @@ func TestTLSCollectingProcess(t *testing.T) {
 	<-cp.GetMsgChan()
 	cp.Stop()
 	assert.NotNil(t, cp.templatesMap[1], "TLS Collecting Process should receive and store the received template.")
+	// Check if connection has closed properly or not by writing to it
```

Comment:
Here as well, it is better to check if server is closed properly or not in addition to the connection.

Reply:
Makes sense. Updated. Thanks!